scripts/workflows/wf.yml

```diff
@@ -0,0 +1,78 @@
+steps:
+- id: build-img
+  uses: docker://docker:19.03.10
+  dir: /workspace/scripts/perf/docker
+  args: [build, --tag=spdk, .]
+  
+- id: prepare
+  uses: docker://spdk
+  skip_pull: true
+  runs: [bash, -ex, -c]
+  args:
+  - |
+    nvme format /dev/nvme0 --ses=2 -n=1
+    echo 512 > /proc/sys/vm/nr_hugepages
+    sysctl -w vm.nr_hugepages=512
+  
+- id: run-read-benchmark
+  uses: docker://spdk
+  skip_pull: true
+  runs: [bash, -ex, -c]
+  args:
+  - |
+    export RW=read IOD=256
+    fio scripts/workflows/precondition.fio
+
+    export RW=randread IOD=256 MIX=100
+    LD_PRELOAD=/root/spdk/examples/bdev/fio_plugin/fio_plugin \
```

Comment:
does this work for the subsequent `fio` invocation as well?

Reply:
No, each time I run `fio` with the `spdk_bdev` engine I have to invoke and define `LD_PRELOAD`, it was the only way it worked

Reply:
oh, I see. makes sense